src/canmatrix/formats/cmjson.py

```diff
@@ -48,9 +48,10 @@ def dump(db, f, **options):
     else:
         mode = 'wb'
 
-    additionalFrameColums = []  # type: typing.List[str]
-    if "additionalFrameAttributes" in options and options["additionalFrameAttributes"]:
-        additionalFrameColums = options["additionalFrameAttributes"].split(",")
+    if options.get("additionalFrameAttributes",False):
```

Comment:
please add space after comma (PEP8), or even simplify to single line
```
additionalFrameColums = options.get("additionalFrameAttributes", "").split(",")
```
It should be enough, because `"".split(",")` is exactly `[]`

Reply:
eeeeeee sorry, the `"".split(",")` is NOT `[]` but `[""]` so most of my comments are WRONG!

Reply:
I was thinking about how to write it on single line, so that the code is more condensated. Maybe this:
```python
additionalFrameColums = [x for x in options.get("additionalFrameAttributes", "").split(",") if x]
```